app/screens/bottom_sheet/index.tsx

```diff
@@ -27,7 +27,7 @@ type Props = {
     componentId: AvailableScreens;
     contentStyle?: StyleProp<ViewStyle>;
     initialSnapIndex?: number;
-    footerComponent?: React.FC<BottomSheetFooterProps>;
+    footerComponent?: React.FC<unknown>;
```

Comment:
why unknown? maybe make it generic? or does not really matter?

Reply:
I couldn't think of a better way to do this. The problem is the following:
- If the footer is a Bottom Sheet footer, it receives the BottomSheetFooterProps, because it will be wrapped by a `BottomSheetFooter` component.
- It not, we don't need the BottomSheetFooterProps, among other things, because trying to use `BottomSheetFooter` outside of a BottomSheet will fail.

Another solution to fix this is to have two different props: "bottomSheetFooter" and "tabletFooter".

That may be more "future proof".